### DISCUSSION TITLE
gobencher failing quite often lately

### DISCUSSION THREAD

**Discussion by Tantalor93** [coredns/coredns#6153] [General]
It seems that gobencher check in CoreDNS pull requests is failing quite often lately, it seems that whole Bencher product provided by Orijitech seems to be failing nowadays as it can be on their dashboard https://dashboard.bencher.orijtech.com/

Also there seems to be a no easy way to report issues to them regarding this check and the check is distributed as closed-source, I would propose removing it from CoreDNS and consider rather look at replacing it with open-source solution built on top of Github Actions for example https://github.com/benchmark-action/github-action-benchmark

**Comment by Tantalor93** [comment]
@chrisohaver is there something you can do about this? Can you disable it? I think gobencher is currently installed as Github application to CoreDNS repository

**Comment by chrisohaver** [comment]
I've suspended it.
